lucille-plugins/lucille-pinecone/src/test/resources/PineconeIndexerTest/two-namespaces.conf

```diff
@@ -0,0 +1,32 @@
+connectors: [
```

Comment:
It looks like the test passes the config directly to the indexer constructor; the config is never used by the Runner itself -- which is fine. This means that the connectors and pipelines elements are not needed by this test, so they can be removed. As a general principle, configs used for testing should be minimal and not include settings that the tests don't need.